src/tensorflow.cc

```diff
@@ -582,8 +582,11 @@ SetStringInputTensor(
       &contiguous_buffer, stream, &cuda_copy);
   if (err != nullptr) {
     RESPOND_AND_SET_NULL_IF_ERROR(response, err);
-    FillStringTensor(
-        tensor, tensor_offset + element_idx, request_element_cnt - element_idx);
+    if (element_idx < request_element_cnt) {
```

Comment:
`element_idx` will always be 0 here right?  Any reason for not treating it as 0 here and in L586?

We can then change the for loop to look like:

```
for (size_t element_idx=0; element_idx < str_list.size(); ++element_idx) {
```

Do you see any reason for not making this change?

Reply:
Agree. But `element_idx` needs to be outside of the for loop.